README.md

```diff
@@ -27,7 +27,13 @@ buffer names.
 
 ### Installation
 
-Install the `ivy` package from MELPA / GNU ELPA.
+Install the `counsel` package preferably from MELPA. If you prefer to
+install from GNU ELPA, then make sure that the depenedencies -i.e. the
+`Ivy` and `Swiper` packages- are installed from GNU ELPA also (to
```

Comment:
Package names are lowercase: `ivy` and `swiper`.

If the user installs `counsel`, then they don't have to manually install `ivy` and `swiper` first. `counsel` lists `ivy` and `swiper` as dependencies, so they get automatically installed by the package manager.